spec/models/auth_token_spec.rb

```diff
@@ -8,6 +8,11 @@
     assert(result == :created)
   end
 
+  it 'generates a value for the token' do
+    result = AuthToken.create(access_code: AccessCode.find(1))
+    assert(!result.nil?)
```

Comment:
We prefer to use `expect` over `assert`, so this line should be `expect(result).to_not be nil`. I think that reads more naturally.